### PR TITLE
REL-2226: Add CFHT as a new exchange request type

### PR DESCRIPTION
**Pull request by cquiroz**
This PR lets both classical and queue proposal to get requests from CFHT

### CODE REVIEW

File: bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala [hunk 1]
```diff
@@ -889,9 +889,7 @@ class PartnerView extends BorderPanel with BoundView[Proposal] {view =>
 
         // Hide CFHT for queue proposals
```
Comment (swalker2m) [inline bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala]: I suppose this comment is out of date then?

**Comment by cquiroz** [comment]
Actually it could be simplified even further, I updated the commit to reflect this

**Comment by swalker2m** [comment]
:+1:

File: bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala [hunk 2]
```diff
@@ -887,16 +887,6 @@ class PartnerView extends BorderPanel with BoundView[Proposal] {view =>
...
 
-        // Hide CFHT for queue proposals
-        m.map {
-          case _:QueueProposalClass     =>
-            new DefaultComboBoxModel(PartnerType.values.filterNot(_ == ExchangeCFHT).toArray)
-          case _:ClassicalProposalClass =>
-            new DefaultComboBoxModel(PartnerType.values.toArray)
-          case _                        =>
-            this.peer.getModel
-        }.map(this.peer.setModel)
```
Comment (tpolecat) [inline bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala]: Use `foreach` if you're going to side-effect; this ensures that it happens now and exactly once. `map` doesn't guarantee either.

Comment (cquiroz) [inline bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala]: good hin

**Pull request merged**